src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws DukeException{
```

Comment:
You should catch the DukeException, if not your program would stop running due to the exception.